scripts/gitlab/build_and_test.sh

```diff
@@ -38,12 +39,21 @@ then
 
     if [[ -d /dev/shm ]]
     then
-        prefix="/dev/shm/${hostname}/${spec// /_}"
+        if [[-z ${job_unique_id} ]]; then
```

Comment:
Bash is hard, I know...
I found this in the logs:
```
scripts/gitlab/build_and_test.sh: line 42: [[-z: command not found
```
```suggestion
        if [[ -z ${job_unique_id} ]]; then
```